go.mod

```diff
@@ -19,16 +19,16 @@ require (
 	github.com/goccy/go-yaml v1.9.8
 	github.com/google/go-containerregistry v0.13.0
 	github.com/mholt/archiver/v3 v3.5.1
-	github.com/moby/moby v20.10.23+incompatible
+	github.com/moby/moby v23.0.1+incompatible
```

Comment:
This was exploding on 23 for me before, assume this is the helm/oras replace stuff?

Reply:
Correct I patched oras to work on this version of moby